superset/sqllab/sql_json_executer.py

```diff
@@ -101,6 +102,7 @@ def execute(
             raise
         except Exception as ex:
             logger.exception("Query %i failed unexpectedly", query_id)
+            logger.error(ex)
```

Comment:
Is this to get some message from the exception logged?

Reply:
`logger.exception` already logs the exception, in addition to the message, it's what differentiates it from `logger.error`. So this line is not needed.